daemon/daemon.go

```diff
@@ -116,7 +116,7 @@ func (c *Command) canAccess(r *http.Request) bool {
 
 func (c *Command) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if !c.canAccess(r) {
-		Forbidden("access denied").ServeHTTP(w, r)
+		Unauthorized("Unauthorized").ServeHTTP(w, r)
```

Comment:
Can we please preserve the "access denied" message here? Although undesired, finding "error: access denied" still feels better than e.g. "error: Unauthorized".